compiler/rustc_codegen_gcc/config.sh

```diff
@@ -0,0 +1,48 @@
+set -e
+
+export CARGO_INCREMENTAL=0
+
+export GCC_PATH=$(cat gcc_path)
+
+unamestr=`uname`
+if [[ "$unamestr" == 'Linux' ]]; then
+   dylib_ext='so'
+elif [[ "$unamestr" == 'Darwin' ]]; then
+   dylib_ext='dylib'
+else
+   echo "Unsupported os"
+   exit 1
+fi
+
+HOST_TRIPLE=$(rustc -vV | grep host | cut -d: -f2 | tr -d " ")
+TARGET_TRIPLE=$HOST_TRIPLE
+#TARGET_TRIPLE="aarch64-unknown-linux-gnu"
+
+linker=''
+RUN_WRAPPER=''
+if [[ "$HOST_TRIPLE" != "$TARGET_TRIPLE" ]]; then
+   if [[ "$TARGET_TRIPLE" == "aarch64-unknown-linux-gnu" ]]; then
+      # We are cross-compiling for aarch64. Use the correct linker and run tests in qemu.
+      linker='-Clinker=aarch64-linux-gnu-gcc'
+      RUN_WRAPPER='qemu-aarch64 -L /usr/aarch64-linux-gnu'
+   else
+      echo "Unknown non-native platform"
+   fi
+fi
+
+export RUSTFLAGS=$linker' -Cpanic=abort -Cdebuginfo=2 -Zpanic-abort-tests -Zcodegen-backend='$(pwd)'/target/'$CHANNEL'/librustc_codegen_gcc.'$dylib_ext' --sysroot '$(pwd)'/build_sysroot/sysroot'
+#export RUSTFLAGS=$linker' -Cpanic=abort -Cdebuginfo=2 -Zpanic-abort-tests -Zcodegen-backend='$(pwd)'/target/'$CHANNEL'/librustc_codegen_gcc.'$dylib_ext' --sysroot '$(pwd)'/build_sysroot/sysroot -Clto=fat -Cembed-bitcode=yes'
+
+# FIXME remove once the atomic shim is gone
```

Comment:
```suggestion
# FIXME(antoyo) remove once the atomic shim is gone
```

Reply:
Technically this was copied from cg_clif, which is written by me. This is no longer necessary for cg_clif, but seemingly it still is for cg_gcc. I don't think there is much value attributing this FIXME to anyone, as someone else may fix it anyway.

Reply:
For this isn't it relevant to know who is originating the note for a fix or further work? I think the git blame would attribute the notice but it's possible to still be lost on copy or future file changes. It doesn't seem that hard to 'sign' these flags as it could be useful for a future reader lacking context.